### PR TITLE
[#2604][#2606] improve(IT): add more test cases for e2e test of metalake page

### PR DESCRIPTION
**Pull request by ch3yne**
### What changes were proposed in this pull request?

Add test cases for end-to-end testing of the Metalake webpage.
- Refresh page
- Check`Datastrato` link
- Check `License` link
- Check `Documentation` link
- Check `Support` link

### Why are the changes needed?

Fix: #2604 
Fix: #2606

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

IT/e2e

o2

### CODE REVIEW

**Comment by LauraXia123** [comment]
LGTM, Thanks @ch3yne

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/MetalakePageTest.java [hunk 1]
```diff
@@ -113,4 +113,53 @@ public void testLinkToCatalogsPage() throws InterruptedException {
...
+    Assertions.assertTrue(
+        metalakePage.verifyLinkInNewWindow(originalWindowHandle, expectedUrl, false));
+  }
+
+  @Test
+  @Order(12)
+  public void testCheckLinkDocs() {
+    String originalWindowHandle = driver.getWindowHandle();
+    metalakePage.footerLinkDocs.click();
+    String expectedUrl = "https://datastrato.ai/docs/";
```
Comment (xunliu) [inline integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/MetalakePageTest.java:151]: Please define static final variable in the class head and replace this, maybe like this
```
private static final String FOOTER_LINKE_DOCS = "https://datastrato.ai/docs/";
```

and another places.

File: integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java [hunk 2]
```diff
@@ -297,6 +317,61 @@ public boolean verifyLinkToCatalogsPage(String name) {
...
+  public boolean verifyRefreshPage() {
+    try {
+      WebDriverWait wait = new WebDriverWait(driver, MAX_TIMEOUT);
+      wait.until(
+          webDriver ->
+              ((JavascriptExecutor) webDriver)
+                  .executeScript("return document.readyState")
+                  .equals("complete"));
+      String pageTitle = metalakePageTitle.getText();
+      boolean isPageTitle = pageTitle.equals("Metalakes");
```
Comment (xunliu) [inline integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/MetalakePage.java:334]: Please define static final variable in the class header to replace `Metalakes`.

**Review by LauraXia123 — APPROVED** [review]

**Review by xunliu — APPROVED** [review]
LGTM

**Pull request merged**